test/ninja/build_libraries.py

```diff
@@ -90,12 +74,16 @@
 
 # clean build and ninja files
 test.run(arguments='-c', stdout=None)
-test.must_contain_all_lines(test.stdout(), [
-    ('Removed %stest_impl' % lib_prefix) + lib_suffix,
-    'Removed test' + _exe,
-    ('Removed %stest_impl_static' % lib_prefix) + staticlib_suffix,
-    'Removed test_static' + _exe,
-    'Removed build.ninja'])
+test.must_contain_all_lines(
+    test.stdout(),
+    [
+        ('Removed %stest_impl' % dll_) + _dll,
```

Comment:
does _dll end up with .so on non-win32?

Reply:
Yes, `_dll` is set fo `dll_suffix`, which can be:

```
    dll_suffix   = '.dll'
    dll_suffix   = '.dll'
    dll_suffix   = '.so'
    dll_suffix   = '.dylib'
    dll_suffix   = '.so'
    dll_suffix   = '.so'
```

for the various platforms, as set in `testing/framework/TestCommon.py`